examples/bri-2/commit-mgr/src/blockchain/utils.js

```diff
@@ -110,3 +109,31 @@ export const jsonrpc = async (method, params, id) => {
   });
   return response.data;
 }
+
+export const chainName = () => {
+  let name;
+  switch (parseInt(process.env.CHAIN_ID, 10)) {
+    case 1:
+      name = 'MAINNET';
+      break;
+    case 3:
+      name = 'ROPSTEN';
+      break;
+    case 4:
+      name = 'RINKEBY';
+      break;
+    case 5:
+      name = 'GOERLI';
+      break;
+    case 42:
+      name = 'KOVAN';
+      break;
+    case 101010:
```

Comment:
Seems a bit redundant. If you remove the `case 101010` block and leave the CHAIN_ID as `101010` the result is the same.